cli/src/main/java/de/jplag/cli/server/ContentType.java

```diff
@@ -0,0 +1,39 @@
+package de.jplag.cli.server;
+
+/**
+ * The type of data
+ */
+public enum ContentType {
+    HTML("text/html; charset=utf-8"),
+    JS("application/javascript; charset=utf-8"),
+    CSS("text/css; charset=utf-8"),
+    PNG("image/png"),
+    PLAIN("text/plain; charset=utf-8"),
+    ZIP("application/zip");
+
+    private final String value;
+
+    ContentType(String value) {
+        this.value = value;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    /**
+     * Guesses the type from the given path using the suffix after the last '.'.
+     * @param path The path to guess from
+     * @return The guessed type
+     */
+    public static ContentType fromPath(String path) {
+        return switch (path.substring(path.lastIndexOf('.'))) {
```

Comment:
No switch is required if you make the suffix a field of the numerals.
Then you can just iterate over all and check which one fits. Thus easier to maintain and to extend.